src/components/BookContainer.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Navi from './nav';
+import BookList from './BookList';
+import InputBook from './InputBook';
+
+const BookContainer = () => (
+  <main>
+    <Navi />
+    <BookList />
+    <div>
+      <BookList />
```

Comment:
[OPTIONAL]

- I would recommend removing one of the components. Since it's just printing booklist two times which doesn't make sense here.

Reply:
Dear @Dipeshtwis 🥇 

I have fixed all you marked. Thanks for your review 👍 

My regards,